client/controllers/incidentReports/incidentForm.coffee

```diff
@@ -59,6 +60,12 @@ Template.incidentForm.onCreated ->
 
     @incidentStatus.set(@incidentData.status or '')
 
+    @locations.set(@incidentData.locations.map (loc)->
+      id: loc.id
+      text: formatLocation(loc)
+      item: loc
+    )
```

Comment:
I think the they make this a little more readable. In general, I don't think we should worry about whether parenthesis are used.